public/index.html

```diff
@@ -39,8 +39,9 @@
       }
     </script> -->
     <!--海岸画像-->
-    <img src="assets/cleanbeach2_img.PNG" id="beach">
-    <img src="assets/cleanbeach1_img.PNG" id="beach">
-    <img src="assets/dirtybeach_img.PNG" id="beach">
+    <div class="beach">
+    <img src="assets/dirtybeach_img.PNG" class="beach0">
+    <img src="assets/cleanbeach1_img.PNG" class="beach1">
+    <img src="assets/cleanbeach2_img.PNG" class="beach2">
```

Comment:
divタグを閉じた方が良さそうです
あとインデントも調整してもらえるとさらに見やすくなりそうです 👍 
```suggestion
    <div class="beach">
      <img src="assets/dirtybeach_img.PNG" class="beach0">
      <img src="assets/cleanbeach1_img.PNG" class="beach1">
      <img src="assets/cleanbeach2_img.PNG" class="beach2">
    </div>
```

Reply:
修正しました